collector/mdadm_test.go

```diff
@@ -0,0 +1,34 @@
+package collector
+
+import (
+	"testing"
+)
+
+func TestMdadm(t *testing.T) {
+	mdStates, err := parseMdstat("fixtures/mdstat")
+
+	if err != nil {
+		t.Error("parsing of reference-file failed entirely")
+		t.FailNow()
```

Comment:
Oh, I didn't even know that `FailNow()` existed. It's more common to just do `t.Fatal(...)` / `t.Fatalf(...)` above, which will log and do the same thing.